### ISSUE TITLE
Update PIDFile= in mbpfan.service

### ISSUE THREAD

**Issue by jeremiah-miller** [linux-on-mac/mbpfan#248]
I ran into this today https://bbs.archlinux.org/viewtopic.php?id=243455 I noticed my computer was warm and the fan didn't run so I ran ran systemctl status mbpfan.service and saw the same error as described here. Unfortunately I already rebooted so I don't have the exact error to copy

**Comment by jeremiah-miller** [comment]
```
Mar 02 17:23:13 jerry systemd[1]: Started A fan manager daemon for MacBook Pro.
Mar 03 08:53:46 jerry systemd[1]: /usr/lib/systemd/system/mbpfan.service:10: PIDFile= references a path below legacy directory /var/run/, updating /var/run/mbpfan.pid → /run/mbpfan.pid; please update the unit file accordingly.
```

**Comment by gaul** [comment]
Fixed by #252.
